python/mlc_chat/quantization/group_quantization.py

```diff
@@ -9,7 +9,9 @@
 from tvm.runtime import NDArray
 from tvm.target import Target
 
+from mlc_chat import op as op_ext
 from mlc_chat.loader import QuantizeMapping
+from mlc_chat.model.mixtral.mixtral_model import MixtralExperts
```

Comment:
It creates a potential cyclic dependency. Currently `mlc_chat.model` depends on `mlc_chat.quantization`, and this line adds a backward edge from `mlc_chat.quantization` to `mlc_chat.model` that makes it a cycle

Reply:
To avoid cyclic dependency, I would suggest that we have a standalone package `mlc_chat.nn` that defines extra common `nn.Module`s used in MLC, such as `MixtralExperts`